src/main/kotlin/com/tailoredapps/gradle/localize/extension/BaseLocalizeExtension.kt

```diff
@@ -12,7 +12,7 @@ import org.gradle.api.NamedDomainObjectContainer
  * @param addComments Whether the comments from the spreadsheet should be added to the strings.xml files (as comments)
  * as well.
  * @param escapeApostrophes Whether apostrophes should be escaped.
- * @param generateEmptyValues Whether empty [values] should be put into the `strings.xml`. Defaults
+ * @param generateEmptyValues Whether empty [generateEmptyValues] should be put into the `strings.xml`. Defaults
```

Comment:
Was wrongly put into `[]`, but I guess not the parameter but the _keyword_ values is meant here, so:

```
@param generateEmptyValues Whether empty `values` should be put into the `strings.xml`.
```

Reply:
done